README.md

```diff
@@ -29,13 +31,7 @@ Begin by creating a configuration file for your application:
 cp .env.example .env
 ```
 
-Edit `.env` with the four configuration parameters we gathered from above. Export
-the configuration in this file as system environment variables like so on Unix
-based systems:
-
-```bash
-source .env
-```
+Edit `.env` with the five configuration parameters we gathered from above.
```

Comment:
we should include a step before this to include setting up a virtual environment